src/components/UserProfile/UserProfile.jsx

```diff
@@ -308,6 +309,7 @@ function UserProfile(props) {
       const response = await axios.get(ENDPOINTS.USER_PROFILE(userId));
       const currentUserEmail = response.data.email;
       dispatch(setCurrentUser({ ...props.auth.user, email: currentUserEmail }));
+      setCanResetPassword(response.data.permissions.frontPermissions.includes('resetPassword'));
```

Comment:
```suggestion
      const canResetPassword = props.hasPermission('resetPassword');
```
`hasPermission()` checks both role and individual permissions. The current way only checks individual permissions. Doesn't need to be a state obj because it won't change after being set.

Reply:
Changed to use `updatePassword` permission instead.